client/stores/user.ts

```diff
@@ -14,17 +18,25 @@ type PretendingToBe = {
   pretendingToBe: number | null
 }
 
+type State = ProfileData & PretendingToBe
+
+const getCurrentRelativePath = (): string => {
+  const locationStr = location.toString() // stringify the whole URL including path, query params, hash
+  const relativePath = locationStr.substring(locationStr.indexOf(location.host) + location.host.length)
+  return relativePath
+}
+
 export const useUserStore = defineStore('user', {
   state: () => {
-    const defaultState: ProfileData & PretendingToBe = {
-      authenticated: false,
+    const defaultState: State = {
+      authenticated: undefined,
       id: null,
-      name: 'Guest',
+      name: '',
       email: '',
       avatar: '',
       rpcPersonId: null,
       isManager: false,
-      pretendingToBe: null // demo/debug only!
+      pretendingToBe: null // demo/debug only! FIXME: disallow on prod?
```

Comment:
We can probably do away with it sooner than that - it had a limited scope to start with and hasn't been maintained. People can just log in as other real users now so I think it's outlived its usefulness.